sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -1063,118 +1157,110 @@
                   "components": [
                     {
                       "label": "Date of birth",
-                      "format": "yyyy-MM-dd",
-                      "tableView": false,
-                      "enableMinDateInput": false,
-                      "datePicker": {
-                        "maxDate": "moment()",
-                        "disableWeekends": false,
-                        "disableWeekdays": false,
-                        "showWeeks": true,
-                        "startingDay": 0,
-                        "initDate": "",
-                        "minMode": "day",
-                        "maxMode": "year",
-                        "yearRows": 4,
-                        "yearColumns": 5,
-                        "minDate": null
-                      },
-                      "enableMaxDateInput": true,
-                      "enableTime": false,
-                      "validate": {
-                        "required": true,
-                        "custom": "",
-                        "customPrivate": false,
-                        "strictDateValidation": false,
-                        "multiple": false,
-                        "unique": false
-                      },
-                      "key": "dateOfBirth",
-                      "type": "datetime",
-                      "input": true,
+                      "labelPosition": "top",
```

Comment:
These changes are with respect to datepicker changes I made. (SIMS #1290)

Reply:
:)